### DISCUSSION TITLE
Some styles do not take effect with the correct configuration

### DISCUSSION THREAD

**Discussion by cnsource** [tailwindlabs/tailwindcss#13406] [Help] [answered]
I found a bug-like issue with tailwindcss in vite, in the 01.vue of the following demo file, I styles are only partially working, and I'm sure that the correctly configured tailwind.config.js configuration file.
The above question was translated courtesy of DeepL and I'd like to hear back from the Tailwind CSS team!

[demo.zip](https://github.com/tailwindlabs/tailwindcss/files/14811010/demo.zip)

**Comment by cnsource** [comment]
![image](https://github.com/tailwindlabs/tailwindcss/assets/22205879/5905a0a1-acee-46a0-a2fe-a833b42dbc76)
![image](https://github.com/tailwindlabs/tailwindcss/assets/22205879/80d28bae-5b5c-4157-a43d-045822951933)


![image](https://github.com/tailwindlabs/tailwindcss/assets/22205879/ad9061d7-f0cb-40a9-8903-586adbcbae3b)


![image](https://github.com/tailwindlabs/tailwindcss/assets/22205879/6ab88d16-e7e3-4478-af39-c3b9c5799de9)

![image](https://github.com/tailwindlabs/tailwindcss/assets/22205879/2952647f-e19a-4f0d-a66e-ee40b95d4277)

![image](https://github.com/tailwindlabs/tailwindcss/assets/22205879/2906776e-e25e-4afb-b4c3-fa41c4a8f771)

![image](https://github.com/tailwindlabs/tailwindcss/assets/22205879/b33a4455-fb82-4996-a133-855c118125ce)

**Comment by alex-krasikau** [comment, marked as answer]
Hey @cnsource!

It looks like the problem is caused by an incorrect configuration of the Tailwind `content` property. Currently, `./src/*.{vue,css,js}` only captures files in the root folder. To scan files in the `src/components` folder, try updating it to `./src/**/*.{vue,css,js}`. This will ensure that deeper files are included in the scan.

**Reply by cnsource** [reply]
Thank you very much, this is indeed what caused my problem
